topologies/proto/binding.proto

```diff
@@ -27,6 +27,18 @@ message Binding {
   Options options = 3;
 }
 
+// Device Config to be passed during Device binding
+message Configs {
+  // Raw device config
+  repeated bytes cli_config_data = 1;
+
+  // Path to file containing raw device config
+  repeated string cli_config_file = 2;
+
+  // Path to file containing gnmi setRequest device config
+  repeated string gnmi_config_file = 3;
```

Comment:
Should be called `gnmi_set_file`. It's not a config but contains the SetRequest.

Reply:
Done!